### DISCUSSION TITLE
2021-12: S3 兼容 Sucks

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#223] [Announcements]
https://xuanwo.io/reports/2021-12/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by sdvcrx** [comment]
查了一下，HTTP Header 里面日期相关的格式必须要是 GMT： https://stackoverflow.com/a/21121453

**Comment by Xuanwo** [comment]
> @sdvcrx
> 查了一下，HTTP Header 里面日期相关的格式必须要是 GMT： https://stackoverflow.com/a/21121453

有道理，不过很多服务的实现比较宽松，这一波可能是我怪错人了
